src/intl/en/page-wallets.json

```
@@ -1,20 +1,20 @@
 {
-  "page-wallets-accounts-addresses": "Wallets, accounts, and addresses",
+  "page-wallets-accounts-addresses": "Wallets, accounts, keys and addresses",
   "page-wallets-accounts-addresses-desc": "It's worth understanding the differences between some key terms.",
-  "page-wallets-accounts-ethereum-addresses": "An Ethereum account has an Ethereum address, like an inbox has an email address. You can use this to send funds to an account.",
+  "page-wallets-accounts-ethereum-addresses": "An Ethereum account has an address, like an inbox has an email address. This is used to identify your digital assets.",
   "page-wallets-alt": "Illustration of a robot with a vault for a body, representing an Ethereum wallet",
-  "page-wallets-ethereum-account": "An Ethereum account is an entity that can send transactions and has a balance.",
+  "page-wallets-ethereum-account": "An Ethereum account is really a pair of keys - one becomes an address that you can share freely and the other must be kept secret and can be used to make signatures. Together, these keys allow you to hold assets and make transactions.",
```

```suggestion
  "page-wallets-ethereum-account": "An Ethereum account is a pair of keys. One key is the address you can share freely, and the other key you need to keep secret because it's used to sign things. Together, these keys let you hold assets and make transactions.",
```

---

It's important not to equate key and address. There's an additional step that derives an address from the public key.
```suggestion
  "page-wallets-ethereum-account": "An Ethereum account is a pair of keys. One key is used to create the address you can share freely, and the other key you need to keep secret because it's used to sign things. Together, these keys let you hold assets and make transactions.",
```